virt/release_notes/virt-4-14-release-notes.adoc

```diff
@@ -164,7 +164,7 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 //CNV-28944 Release note: Preview Cluster level eviction strategy change
 
 //CNV-29940 Release note: Preview UI Bridged network interface hot-plug for VMs
-* You can add a bridged network interface hot-plug to a running VirtualMachine and either live-migrate or restart the VM to apply the changes. Cluster administration permissions are required. 
+* You can add a bridged network interface hot-plug to a running VirtualMachine and either live migrate or restart the VM to apply the changes. Cluster administration permissions are required. 
```

Comment:
"Hot-plug" can be used as a verb. "You can hot-plug a bridged network interface to a ..."

There are conditions under which the live migration step isn't necessary: https://kubevirt.io/user-guide/operations/hotplug_interfaces/ (Requirements section). 
I'm not sure if you omitted that on purpose, but I wanted to make sure you're aware in case it needs to be added.

To my knowledge, cluster admin permissions are only necessary to enable the feature.